core/templates/domain/question/QuestionObjectFactory.ts

```diff
@@ -17,9 +17,13 @@
  * question domain objects.
  */
 
-require('domain/state/StateObjectFactory.ts');
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { Injectable } from '@angular/core';
+import { State, StateBackendDict, StateObjectFactory }
+  from 'domain/state/StateObjectFactory';
 
-import { StateBackendDict } from 'domain/state/StateObjectFactory';
+const INTERACTION_SPECS = require('interactions/interaction_specs.json');
+const constants = require('constants.ts');
```

Comment:
The import order we follow is this:

1. Angular imports (imports from @angular/*)
2. Third-party imports (imports from rxjs, ng bootstrap, etc)
3. Application imports. (imports from our application)

Within these sub-categories: sort lexicographically based on module (i.e. where it is imported from, not what is being imported).
For example:
#### Right way (because 'B' < 'D')
```
import { C } from 'B';
import { A } from 'D'; 
```

#### Wrong way
```
import { A } from 'D';
import { C } from 'B'; 
```

Can you change that here and everywhere else in this PR (only for the migrated files)?

Reply:
Fixed